backend/benefit/applications/migrations/0017_searchable_encrypted_social_security_number.py

```diff
@@ -0,0 +1,31 @@
+# Generated by Django 3.2.4 on 2021-09-03 11:07
+
+from django.db import migrations
+import encrypted_fields.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("applications", "0016_benefit_amount"),
+    ]
+
+    operations = [
+        migrations.RenameField(
+            model_name="employee",
+            old_name="social_security_number",
+            new_name="encrypted_social_security_number",
+        ),
+        migrations.AddField(
+            model_name="employee",
+            name="social_security_number",
+            field=encrypted_fields.fields.SearchField(
+                blank=True,
+                db_index=True,
+                encrypted_field_name="encrypted_social_security_number",
+                hash_key="ee235e39ebc238035a6264c063dd829d4b6d2270604b57ee1f463e676ec44669",
```

Comment:
Yes, fixed to use settings.SOCIAL_SECURITY_NUMBER_HASH_KEY here, as we probably should
have different value in staging/prod. Although the value in migration doesn't probably matter as it's not part of db schema, it might be confusing.